dev/docker/env

```diff
@@ -2,5 +2,5 @@
 set -e
 
 function docker_compose() {
-  docker-compose -f dev/docker/docker-compose.yml -p xmtpd "$@"
+  docker compose -f dev/docker/docker-compose.yml -p xmtpd "$@"
```

Comment:
Apparently our Github runners have been upgraded to a new version where `docker-compose` no longer exists as a separate command. You must use `docker compose` instead.